### ISSUE TITLE
Error in Running WIT tool in AWS Sagemaker Studio

### ISSUE THREAD

**Issue by sanketcn** [PAIR-code/what-if-tool#146]
When I tried to run WIT tool in Aws sagemaker Studio I am facing 3 issues
1. 
   !jupyter labextension install wit-widget
    !jupyter labextension install @jupyter-widgets/jupyterlab-manager --yes
     This step takes too much time (sometimes it got stuck for hours)
2.  Error displaying widget (when running the cell to show WIt tool)
3.  For large database it got stuck on cell which is going to show WIT tool

Is there any other requirements to run the WIT tool?

**Comment by jameswex** [comment]
Thanks for reaching out. Haven't tried WIT in Sagemaker Studio before. I'll give it a shot and let you know what I find.

**Comment by jameswex** [comment]
@sanketcn I'm new to Sagemaker Studio. When I try "!jupyter labextension install wit-widget" in a notebook cell after creating a user and opening the studio, I get an error to "Please install Node.js and npm before continuing installation". What steps do I need to take from a fresh studio instance to repro your issues? Thanks.

**Comment by sanketcn** [comment]
@jameswex  you can use this command to get rid of this error
!conda install nodejs --yes
Hope it helps.

**Comment by jameswex** [comment]
Thanks. After that installation, my "!jupyter labextension install wit-widget" errors with "PermissionError: [Errno 13] Permission denied: '/opt/conda/envs/sagemaker-soln/share/jupyter/lab/extensions'".

Have you seen that before?

I then tried "!sudo jupyter labextension install wit-widget" but get errors:
sudo: unable to resolve host sagemaker-jumpstart-d-ml-t3-medium-6375f6791df4e5acd4cf09378a18
sudo: jupyter: command not found

**Comment by sanketcn** [comment]
No,I have not seen that error during installation but I found some issue on github regarding jupyterlabextensions
https://github.com/Calysto/matlab_kernel/issues/68

can you please try this
!jupyter labextension install wit-widget --user"

**Comment by pri2si17-1997** [comment]
Hi @sanketcn @jameswex did you find any solution to run it?

**Comment by jameswex** [comment]
Sorry I haven't had the bandwidth to investigate this. In general you may wish to instead check out our LIT tool which is a more customizable and extensible tool than WIT. https://pair-code.github.io/lit/
